content/blog/2023-11-28-cis-benchmarks-update.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "CIS Benchmarks Update"
+type: blog
+author:
+ name: "Simon John"
+ bio: "Security Standards Architect"
+ image: /users/sjohn.png
+date: '2023-11-28'
+post:
```

Comment:
```suggestion
images:
  - /blog-images/cis_update.svg
post:
```

Reply:
Added this from our updated template. This is the image that gets shared when the link is posted to social media.

Reply:
Ah right I wondered why we got the anaconda image all the time on LinkedIn etc